ComposeSamples/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
     }
     dependencies {
         classpath config.gradlePlugin
-        classpath config.kotlinGradlePlugin
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.32"
```

Comment:
Not sure if this slipped into the files committed (using the hardcoded name and version instead of referring to the `dependencies.gradle` file)

Reply:
Good catch, updated, thanks!